src/annotator/util/test/listener-collection-test.js

```diff
@@ -0,0 +1,30 @@
+import { ListenerCollection } from '../../util/listener-collection';
+
+describe('ListenerCollection', () => {
+  let listeners;
+
+  beforeEach(() => {
+    listeners = new ListenerCollection();
+  });
+
+  afterEach(() => {
+    listeners.removeAll();
+  });
+
+  it('registers and triggers event listener', () => {
+    const listener = sinon.stub();
+    listeners.add(window, 'resize', listener);
+
+    window.dispatchEvent(new Event('resize'));
+    assert.calledOnce(listener);
+  });
+
+  it('unregister event listeners', () => {
```

Comment:
```suggestion
  it('unregisters event listeners', () => {
```

Reply:
Hi Eduardo - This was marked as resolved but the suggestion wasn't applied.

Reply:
Sorry, for that. I fixed that.